### ISSUE TITLE
training from scratch can't get proper model

### ISSUE THREAD

**Issue by Wei-Baldwin-Zeng** [facebookresearch/nougat#196]
Great work! 

I'm now trying to **train nougat from scratch**. 

**The data** I use is generated from arxiv, as the script the author provided,  I got **130w** in total for now. 
**The machine** I use is one machine with **8 v100**. The batch size is 1, to be aligned with the original paper, I changed the **accumulate_grad_batches to 24**, so the  effective batch size is **1*8*24=192**. 

however, the training loss is saturated around 1, I have tried the provided checkpoint, to get reasonable results, the loss should be round 0.05, right?

**Can you share your training log or training loss curve?** 

Is there anything I should pay attention, to get proper model training from scratch? @lukas-blecher 

Best regards,

**Comment by sgdescent** [comment]
Hey,
Did you pretrain the model? If not I would suspect that to be an issue. The paper mentions some form of pre-training for basic OCR recognition

**Comment by Wei-Baldwin-Zeng** [comment]
> Hey, Did you pretrain the model? If not I would suspect that to be an issue. The paper mentions some form of pre-training for basic OCR recognition

Is the model just using the general pretrained weights of swin transformer and mBart, and directly trained end-to-end on the arxiv data (and PMC+IDL data, which I didn't use for now)?
